build/networking/scripts/build_p4c.sh

```diff
@@ -17,7 +17,6 @@ fi
 P4C_SHA=4837beadf3fc390dd3a39434fec4fa6658e993be
```

Comment:
I think you need to update this as well or we'll be using a really ancient version of p4c during the build.

Reply:
done